spec/parser_spec.lua

```diff
@@ -219,65 +228,83 @@ describe("Pallene parser", function()
         assert_program_ast([[
             local function fA() : float
             end
-        ]], {
-            { _tag = "ast.Toplevel.Func",
-                visibility = "local",
-                decl = {
-                    name = "fA",
-                    type = {
-                        arg_types = {},
-                        ret_types = {
-                            { _tag = "ast.Type.Float" }, }, } },
-                value = {
-                    _tag = "ast.Exp.Lambda",
-                    arg_decls  = {},
-                    body = {
-                        _tag = "ast.Stat.Block",
-                        stats = {} } } },
-        })
+        ]],
+        {{
```

Comment:
Could we keep theses tests indented the way they were before? I'm not sure I like this `{{` at the start -- it doesn't scale well to ASTs where there is more than one toplevel node.

Another problem is that making this kind of change makes the diff harder to read. Generally speaking it is better to put this kind of indentation change in a separate commit/PR.